src/elm/Session/LoggedIn.elm

```diff
@@ -886,8 +1018,22 @@ update msg model =
                                     |> UR.addExt AuthenticationFailed
 
                             Auth.CompletedAuth _ ->
+                                let
+                                    cmd =
+                                        case model.claimingAction.status of
+                                            Action.ClaimInProgress action maybeProof ->
+                                                -- If action claim is in progress,
+                                                -- send a message to finish the claiming process
+                                                -- when the user confirms the PIN.
+                                                Task.succeed (GotActionMsg (Action.ActionClaimed action maybeProof))
+                                                    |> Task.perform identity
```

Comment:
This is the second thing I don't like very much.

To claim the action user has to have a PIN confirmed. Usually, we use `UR.addExt RequiredAuthentication ...` but it doesn't work from the `LoggedIn` module.

Here's how it works:

![image](https://user-images.githubusercontent.com/140053/107627503-74675c80-6c70-11eb-8b6d-e14cf0690e43.png)

Reply:
it makes sense, inside logged in stuff will work differently